src/yaml.ts

```
@@ -0,0 +1,679 @@
+        const validate = (rpcConfig: any, key: string, value: any) => {
+            assert(
+                key === "url" || key === "weight" || key === "trackSize",
+                `/unknown key: ${key}`,
+            );
+            if (key === "url") {
+                assert(!("url" in rpcConfig), "duplicate url");
+                rpcConfig.url = value;
+            }
+            if (key === "weight") {
+                assert(!("selectionWeight" in rpcConfig), "duplicate weight option");
+                const parsedValue = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)